Document decision for feature to omit RBAC creation on org namespaces

TheBigLee opened the pull request:
The documented decision for the new feature implemented in this PR: https://github.com/appuio/appuio-cloud-agent/pull/54
<!--
Thank you for your pull request. Please provide a description above and
review the checklist below.

Contributors guide: ./CONTRIBUTING.md
-->

## Checklist
<!--
Remove items that do not apply. For completed items, change [ ] to [x].
-->

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

<!--
NOTE: these things are not required to open a PR and can be done afterwards,
while the PR is open.
-->

simu reviewed (changes requested):
Actual new documentation page is missing.

Also spelling: "ommit" should be "omit"

TheBigLee commented:
> Actual new documentation page is missing.
> 
> Also spelling: "ommit" should be "omit"

Git is hard :see_no_evil:

Fixed the typo and added the actual document to the PR.

tobru commented on docs/modules/ROOT/pages/explanation/decisions/no_rbac_creation.adoc:
```diff
@@ -0,0 +1,26 @@
+
+== Problem
+
+The "Managed by VSHN" appcat services provision the instances in a separate namespace that has the same organization label as the namespace where the claim originates from. By default, this will give the organization users full access to those managed namespaces and would allow the users to manipulate the managed appcat services in an undesired way.
+To avoid that, the default RBACs should not be created for those namespaces.
```
I recommend not binding that problem to a particular product. I'd rather state that in a more generic way.

If you want to stick with that: appcat -> "VSHN Application Catalog"

TheBigLee replied:
I have rephrased the paragraph. Does it work for you this way?

tobru replied:
Yes, now it makes most sense to me. Thanks.

simu reviewed (approved):
LGTM overall
